lib/en/ceylon-overview.adoc

```diff
@@ -0,0 +1,102 @@
+---
+layout: base
+category: 20_Languages
+breadcrumb: Languages
+parent_url: languages-overview.html
+nav_title: Ceylon
+nav_priority: 51
+meta_desc: Ceylon Developers - OpenShift Resources to get your Ceylon applications hosted in the cloud.
+---
+= Ceylon Application Hosting
+
+[[top]]
+[[ceylon]]
+[float]
+= Ceylon Application Hosting
+[.lead]
+With OpenShift you can easily deploy and run Ceylon applications using your favorite application servers or frameworks. You can run them on WildFly, bare-bones with `ceylon.net` or with Vert.x.
+
+Ceylon is a new programming language that's deeply influenced by Java, designed by people who are unapologetic fans of Java. It's a language designed specifically for writing large programs in teams.
+
+Visit the official Ceylon documentation to link:http://ceylon-lang.org[learn more].
+
+image::ceylon-logo.png["Ceylon logo"]
+
+== Supported Technologies
+OpenShift provides several ways to deploy Ceylon applications, including the following environments:
+
+=== link:wildfly-overview.html[Wildfly]
+OpenShift Online currently supports `Wildfly 8.2 Final`. +
+link:/en/wildfly-overview.html[-> Learn More]
+
+=== link:vertx-overview.html[Vert.x]
+link:http://vertx.io/[Vert.x] is a lightweight, high performance polyglot application platform for the JVM that's designed for modern mobile, web, and enterprise applications. OpenShift Online currently supports `Vert.x 2.1`. +
+link:/en/vertx-overview.html[-> Learn More]
+
```

Comment:
Ah yes, that's because you can either use the dedicated cartridge, or deploy your Ceylon application as a war to the WildFly cartridge, or just as-is to Vert.x which knows how to deal with Ceylon applications. But you're right this is not clear and I should make it clearer. I'll updated my PR tonight, thanks.